uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Service/LeaveRequestRights.php

```diff
@@ -232,9 +232,9 @@ public function canCancelForAbsenceType($absenceTypeId, $contactId, DateTime $le
 
     $today = new DateTime('today');
 
-    if ($absenceType->allow_request_cancelation != AbsenceType::REQUEST_CANCELATION_NO &&
-      $leaveFromDate > $today
-    ) {
+    $absenceTypeRequestCancellationNotNo =
```

Comment:
Sorry to be picky about this but `NotNo` seems like it's possible to be improved, would it be possible to name this something like `absenceTypeAllowsCancellation`?